modin/config/pubsub.py

```diff
@@ -135,11 +135,14 @@ class Parameter(object):
         ``ValueSource``.
     """
 
-    choices: typing.Sequence[str] = None
+    choices: Optional[Sequence[str]] = None
```

Comment:
Should fields like this that are now optional also be reflected in the docstring?

Reply:
@noloerino yes thank you for pointing this out!